hbase-client/src/main/java/org/apache/hadoop/hbase/client/Mutation.java

```diff
@@ -1000,25 +999,16 @@ public byte getTypeByte() {
 
     @Override
     public Optional<Tag> getTag(byte type) {
-      if (cell instanceof RawCell) {
-        return ((RawCell) cell).getTag(type);
-      }
       return PrivateCellUtil.getTag(cell, type);
     }
 
     @Override
     public Iterator<Tag> getTags() {
-      if (cell instanceof RawCell) {
-        return ((RawCell) cell).getTags();
-      }
       return PrivateCellUtil.tagsIterator(cell);
     }
 
     @Override
     public byte[] cloneTags() {
-      if (cell instanceof RawCell) {
-        return ((RawCell) cell).cloneTags();
```

Comment:
Ok so `RawCell.cloneTags()` internally refers `PrivateCellUtil.cloneTags(cell)` only, hence the removal and same is applicable to above methods.

Reply:
Yes.

Trivial change. Just shrinking the use of RawCell so it more obvious it is only needed in one location. Its weird having RawCell out in the client module when internally it is just doing the PrivateCellUtil method anyways.

I think this circumstance came about innocently after going back over git history... This was just not noticed before this.